src/lib/FlightTasks/tasks/FlightTaskManualStabilized.cpp

```diff
@@ -138,10 +126,19 @@ float FlightTaskManualStabilized::_throttleCurve()
 	}
 }
 
-bool FlightTaskManualStabilized::update()
+void
+FlightTaskManualStabilized::initialiseOutputs(ControlSetpoint &setpoint)
```

Comment:
Why is there an extra `initializeOutput`? Activate is called on the even the task needs to initialize itself.

Reply:
naming